.github/workflows/test.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node_version: [10, 12, 13]
+        node_version: [10, 12, 13, 16]
```

Comment:
(This isn't in the scope of this PR)
Is there a particular reason that the next version tested is v13, and not v14 which is the LTS release